### ISSUE TITLE
Wrong description for object properties opEnd and opStart

### ISSUE THREAD

**Issue by MathiasVDA** [Interoperable-data/ERA_vocabulary#16]
The vocabulary describes opStart as:
> Unique OP ID at start of section of line (kilometres increasing from start OP to the end OP). [...]

But the range of the property is an entity of the class "Operational point". Based on the above description, a reader might be confused with the data property 'uopid'.

![image](https://github.com/Interoperable-data/ERA_vocabulary/assets/15101339/b1355b82-b398-4033-9255-932ad3d852eb)

I would suggest to alter the description of opStart and opEnd to something like:
> Operational point at the [...]

**Comment by MathiasVDA** [comment]
This seems to me like the difference between how an ontology is meant to be used, and what the purpose was for the definitions in the legal text. 

For me it's not logically to create an ontology with exactly the legal text and then change the semantic meaning of the concept. In the ontology, the concept is an "object property", in the legal text, the concept is meant as a "data property". Either you change the ontology to correspond to the concepts from the legal text, or you change the descriptions. Keeping both is asking for semantic difficulty.

**Comment by MathiasVDA** [comment]
Yes, so this issue can be used to inform ERA that their guideline doesn't make sense when the concept has been changed to an object property. Besides, my proposed change is still in line with the spirit of the legal text.

**Comment by gatemezing** [comment]
> Yes, so this issue can be used to inform ERA that their guideline doesn't make sense when the concept has been changed to an object property. Besides, my proposed change is still in line with the spirit of the legal text.

We need to reflect that in the new application guide. Please @MathiasVDA don't forget to raise that in our next RINF plenary meeting if needed.

**Comment by DMPatru** [comment]
@MathiasVDA, we will propose this change in the next RINF application guide and in the next revision of the ontology, for a better understanding of the concepts. 
The "RINF Index" annotation property could be used (by those interested) to point to the Table 1 of the RINF Regulation annex.

Same logic will be applied to other properties and classes in the ontology, which do not have a 1-to-1 link with Table 1 rows.
